app/helpers/application_helper.rb

```diff
@@ -27,7 +27,7 @@ def previous_or_next_page(page, text, classname)
   end
 
   def page_navigation_links(pages)
-    will_paginate(pages, class: 'pagination', inner_window: 2, outer_window: 0, renderer: BootstrapLinkRenderer, previous_label: '&larr;'.html_safe, next_label: '&rarr;'.html_safe)
+    will_paginate(pages, class: 'pagination', inner_window: 2, outer_window: 0, renderer: BootstrapLinkRenderer, previous_label: '&larr;'.html_safe, next_label: '&rarr;'.html_safe, page_links: false)
```

Comment:
Thanks for both suggestions. I'll open a new PR for will_paginate and will a second one with a more specific selector string for the other issue.